src/Magento/FunctionalTestingFramework/Console/RunTestGroupCommand.php

```diff
@@ -0,0 +1,104 @@
+<?php
+// @codingStandardsIgnoreFile
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types = 1);
+
+namespace Magento\FunctionalTestingFramework\Console;
+
+use Magento\FunctionalTestingFramework\Suite\Handlers\SuiteObjectHandler;
+use Magento\FunctionalTestingFramework\Test\Handlers\TestObjectHandler;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+class RunTestGroupCommand extends Command
+{
+    /**
+     * Configures the current command.
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        $this->setName('run:group')
+            ->setDescription('Execute a set of tests referenced via group annotations')
+            ->addOption('skip-generate', 'k', InputOption::VALUE_NONE, "only execute a group of tests without generating from source xml")
+            ->addArgument(
+                'groups',
+                InputArgument::IS_ARRAY | InputArgument::REQUIRED,
+                'group names to be executed via codeception'
+            );
+    }
+
+    /**
+     * Executes the current command.
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return void
+     * @throws \Symfony\Component\Console\Exception\LogicException
+     *
+     * @SuppressWarnings(PHPMD.UnusedLocalVariable)
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $skipGeneration = $input->getOption('skip-generate') ?? false;
+        $groups = $input->getArgument('groups');
+
+        if (!$skipGeneration) {
+            $testConfiguration = $this->getGroupAndSuiteConfiguration($groups);
+            $command = $this->getApplication()->find('generate:tests');
+            $args = ['--tests' => $testConfiguration];
+
+            $command->run(new ArrayInput($args), $output);
+        }
+
+        $codeceptionCommand = realpath(PROJECT_ROOT . '/vendor/bin/codecept') . ' run functional --verbose --steps';
+
+        foreach ($groups as $group) {
+            $codeceptionCommand .= " -g {$group}";
+        }
+
+        $process = new Process($codeceptionCommand);
+        $process->setWorkingDirectory(TESTS_BP);
+        $process->run(
+            function ($type, $buffer) use ($output) {
+                $output->write($buffer);
+            }
+        );
+    }
+
+    /**
+     * Returns a json string to be used as an argument for generation of a group or suite
+     *
+     * @param array $groups
+     * @return string
+     */
+    private function getGroupAndSuiteConfiguration(array $groups)
+    {
+        $testConfiguration['tests'] = [];
```

Comment:
I would like them both to be null, but I cannot merge any relevant tests to a value of null. I could do a check for the "first value" and replace instead of merge but this seems like elegant to me. Additionally I could make this command exclusive to test groups and point users to the run suite command if they attempt to run a group that is in fact a suite.